### PR TITLE
add cloud9 instruction for update-behaviors-of-stack-resources

### PR DESCRIPTION
**Pull request by shubham-amaz**
## What does this PR do and why?

add support to cloud9 steps for update-behaviors-of-stack-resources

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/intermediate/operations/update-behaviors-of-stack-resources/index.md [hunk 1]
```diff
@@ -51,6 +51,24 @@ Resources:
...
 
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace/update-behaviors-of-stack-resources`:
+    :::code{language=shell showLineNumbers=false showCopyAction=true}
+    cd code/workspace/update-behaviors-of-stack-resources
+    :::
+    1. Use the AWS CLI to create the stack. The required parameters `--stack-name` and `--template-body` have been pre-filled for you.
+    :::code{language=shell showLineNumbers=false showCopyAction=true}
+    aws cloudformation create-stack --stack-name update-behaviors-of-stack-resources --template-body file://update-behaviors-of-stack-resources.yaml
```
Comment (sdangol) [inline content/intermediate/operations/update-behaviors-of-stack-resources/index.md:62]:
```suggestion
    aws cloudformation create-stack --stack-name cfn-workshop-update-behaviors-of-stack-resources --template-body file://update-behaviors-of-stack-resources.yaml
```

File: content/intermediate/operations/update-behaviors-of-stack-resources/index.md [hunk 2]
```diff
@@ -51,6 +51,24 @@ Resources:
...
+    :::
+    1. If the `create-stack` command was successfully sent, CloudFormation will return `StackId`.
+    :::code{language=shell showLineNumbers=false showCopyAction=true}
```
Comment (sdangol) [inline content/intermediate/operations/update-behaviors-of-stack-resources/index.md:65]:
```suggestion
    :::code{language=shell showLineNumbers=false showCopyAction=false}
```

File: content/intermediate/operations/update-behaviors-of-stack-resources/index.md [hunk 3]
```diff
@@ -51,6 +51,24 @@ Resources:
...
+    "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/drift-detection-workshop/739fafa0-e4d7-11ed-a000-12d9009553ff"
```
Comment (sdangol) [inline content/intermediate/operations/update-behaviors-of-stack-resources/index.md:66]:
```suggestion
    "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-update-behaviors-of-stack-resources/739fafa0-e4d7-11ed-a000-12d9009553ff"
```

File: content/intermediate/operations/update-behaviors-of-stack-resources/index.md [hunk 4]
```diff
@@ -61,6 +79,10 @@ Save your changes to the file. Next, create your stack with the `update-behavior
 8. On the **Review** page, scroll down to the bottom, and choose **Create stack**.
```
Comment (sdangol) [inline content/intermediate/operations/update-behaviors-of-stack-resources/index.md:79]:
```suggestion
8. On the **Review** page, scroll down to the bottom, and choose **Submit**.
```

File: content/intermediate/operations/update-behaviors-of-stack-resources/index.md
Comment (sdangol) [inline content/intermediate/operations/update-behaviors-of-stack-resources/index.md:1]: Missing cloud9 instructions for updating the stack for replacement of parameter

Comment (sdangol) [inline content/intermediate/operations/update-behaviors-of-stack-resources/index.md:1]: Missing cloud9 instructions for updating the sack for "Updates with Some Interruption"

Comment (sdangol) [reply content/intermediate/operations/update-behaviors-of-stack-resources/index.md:1]: No need to implement this after confirming. Resolving.

Comment (sdangol) [reply content/intermediate/operations/update-behaviors-of-stack-resources/index.md:1]: No need to implement this after confirming. Resolving.

**Review by sdangol — DISMISSED** [review]
LGTM

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**